components/Dialogs/tests/unit/MDCAlertControllerTests.m

```diff
@@ -125,4 +125,16 @@ - (void)testAlertControllerColorSetting {
   }
 }
 
+- (void)testAlertControllerSettingTitleAndMessage {
+  NSString *title = @"title";
+  NSString *message = @"message";
+  MDCAlertController *alert = [MDCAlertController alertControllerWithTitle:title
+                                                                   message:message];
+  alert.titleFont = [UIFont systemFontOfSize:25];
+
+  MDCAlertControllerView *view = (MDCAlertControllerView *)alert.view;
+  XCTAssertEqual(view.titleLabel.text, title);
+  XCTAssertEqual(view.messageLabel.text, message);
```

Comment:
What does setting the font affect in this test?

Reply:
it's a property on the view so I am making sure alertView is loaded, that was the breaking scenario.